Add PLATO to Catalogs (#2279)
astropy/astroquery: CHANGES.rst

- Adding moving target functionality to ``astroquery.mast.Tesscut`` [#2121]

> - Adding PLATO catalog as query option in ``astroquery.mast.Catalogs.query_object`` and ``astroquery.mast.Catalogs.query_region``. [#2279]

Reviewer: This is the PLATO input catalogue, right? I would also rephrase a little to concise the entry, but happy to do that before release when I do a changelog cleanup
```suggestion
- Adding the All-Sky PLATO Input Catalog ('plato') as a catalog option for methods of ``astroquery.mast.Catalogs``. [#2279]
```
